flags.css

```diff
@@ -1,4 +1,16 @@
 /* Indonesia */
+.rectangle{
+  height: 90px;
```

Comment:
I would try to do this dynamically.  Since you want the flag to be proportional between its dimensions, you would want to use a similar unit.

So for example:

```
height: 33vmin;
width: 55vmin;
```
Do ask questions about this if you want to know more.

Reply:
And realistically, you would want to apply these traits to the flagContainer, and then make the subpieces be parts of that.

so more like this:
```
.flagContainer{
   width: 50vmin;
   height: 33vmin;
}
#indonesia > .halfRectangle{
   height: 50%;
}
```